docs/Four-Actions.md

```diff
@@ -7,3 +7,12 @@
 | Create An Advantage (Unknown Aspect) | Someone detrimental can reveal it to get the free invoke | The aspect remains unknown, but you get a Boost | Reveal the aspect and get *one* free invoke | Reveal the aspect and get *two* free invokes|
 | Attack | Attack is prevented | No Damage, Attacker Boost | Deal damage equal to shifts | Deal damage, and can trade 1 Shift for a Boost | 
 | Defend | Take damage / Attacker succeeds | Attacker creates a Boost | No Damage / Attacker fails | Prevent damage & *Defender* gains a Boost|
+
+## Weapon and Armor Interactions
+
+[Weapon and Armor](adopted-rules.md) do not impact Succeeding with Style, but their impact on damage can override the normal outcomes for Ties and Success. 
```

Comment:
Nit: this wording might imply that SwS is not affected by weapons and armor, instead, I might just say that Weapon ratings can cause ties to deal damage instead of granting a boost, and Armor ratings can cause successes to deal no damage and create a boost.